server/src/main/java/io/druid/segment/realtime/RealtimeManager.java

```diff
@@ -72,27 +74,34 @@
    */
   private final Map<String, Map<Integer, FireChief>> chiefs;
 
+  private final SegmentManager segmentManager;
```

Comment:
Ah, I added this field for the future use, but it would be fine to add later. I removed for now.